### PR TITLE
test: Unit Test for ClientChargePresenter

### LINKED ISSUE #493: Unit tests for ClientChargePresenter
**Summary:**
Write Unit tests for ClientChargePresenter.

Example for unit testing of <a href="https://github.com/openMF/mifos-mobile/blob/development/app/src/main/java/org/mifos/mobilebanking/presenters/AccountsPresenter.java"> AccountPresenter</a> can be seen in <a href="https://github.com/openMF/mifos-mobile/blob/development/app/src/test/java/org/mifos/mobilebanking/AccountsPresenterTest.java"> AccountsPresenterTest</a>

### PR DESCRIPTION
**Pull request by luckyman20**
Fixes #493

@dilpreet96 Please review it!!

### CODE REVIEW

File: app/src/test/java/org/mifos/mobilebanking/ClientChargePresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        verify(view).showProgress();
+        verify(view).hideProgress();
+        verify(view, never()).showClientCharges(charge.getPageItems());
+        verify(view).
+                showErrorFetchingClientCharges(context.getString(R.string.error_client_charge_loading));
+    }
+
+    @Test
+    public void testLoadLoanAccountCharges() {
+        when(dataManager.getLoanCharges(1)).thenReturn(Observable.just(charge.getPageItems()));
```
Comment (dilpreet2028) [inline app/src/test/java/org/mifos/mobilebanking/ClientChargePresenterTest.java]: You have to use loan charges json, the one which I gave in the mail

File: app/src/test/java/org/mifos/mobilebanking/ClientChargePresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        verify(view).showProgress();
+        verify(view).hideProgress();
+        verify(view, never()).showClientCharges(charge.getPageItems());
+        verify(view).
+                showErrorFetchingClientCharges(context.getString(R.string.error_client_charge_loading));
+    }
+
+    @Test
+    public void testLoadSavingsAccountCharges() {
+        when(dataManager.getSavingsCharges(1)).thenReturn(Observable.just(charge.getPageItems()));
```
Comment (dilpreet2028) [inline app/src/test/java/org/mifos/mobilebanking/ClientChargePresenterTest.java]: Same goes here, use savings charges from the mail

**Review by dilpreet2028 — CHANGES_REQUESTED** [review]
@luckyman20 , make the changes and your travis build is failing please fix it , you can see here the reason for the fail: https://travis-ci.org/openMF/mifos-mobile/builds/314891340#L3087

**Comment by luckyman20** [comment]
@dilpreet96 Updated the PR

**Review by dilpreet2028 — APPROVED** [review]

**Pull request merged**